.gitignore

```diff
@@ -1 +1,4 @@
-node_modules/
\ No newline at end of file
+node_modules/
+tests/
```

Comment:
I add them temporarily. so I can test my code without needing to clean those temporarily files later when I want to commit